src/classiceditorui.js

```diff
@@ -88,6 +99,29 @@ export default class ClassicEditorUI {
 				}
 
 				return Promise.all( promises );
+			} )
+			.then( () => {
+				// Listen on the keystrokes from the main UI.
+				this.keystrokes.listenTo( this.view.element );
+
+				// Listen on the keystrokes from the floating panels, toolbars and the such.
+				this.keystrokes.listenTo( this.view._bodyCollectionContainer );
```

Comment:
I think ` this.view.element` and `this.view._bodyCollectionContainer` should be exposed for plugins to use. 

ATM I'm implementing keystrokes in Link and I found out I'd like to listen for all keystrokes coming from the UI but `document` just feels too wide. `editorui.view#containers` with references to these elements would help because depending on implementation, there could be many other elements in DOM that should support keystrokes.